src/api/comments.js

```diff
@@ -0,0 +1,30 @@
+import { BASE_URL } from './api';
+
+export function getPostComments(postId) {
+  return fetch(`${BASE_URL}/comments/`)
```

Comment:
add parameter to fetch comments for a specific post, not for all posts all together